### DISCUSSION TITLE
Is it possible to add OG:Image metadata to a 404 notfound page?

### DISCUSSION THREAD

**Discussion by PeterGKafkakis** [vercel/next.js#61801] [Help]
### Summary

I wanted to add a 404 OG:image to my not-found page, my title / description metadata work, but when i add the OG:image url it does not show when sharing the link through imessage BUT works for facebook messenger.

My question is, is it even possible to do so in next js 14? 

### Additional information

```js
import { Metadata } from "next";
import { notFound } from "next/navigation";

export const metadata: Metadata = {
  title: "Link Does Not Exist",
  description: "404 Error",
  openGraph: {
    images: [
      {
        url: "test.png",
        alt: "404 Image",
      },
    ],
  },
};

export default function NotFound() {
  notFound();
}
```

**Comment by arunavo4** [comment]
https://github.com/vercel/next.js/discussions/55761
